config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -759,6 +759,33 @@ periodics:
     testgrid-tab-name: gencred-refresh-kubeconfig
     testgrid-alert-email: kubernetes-sig-testing-alerts@googlegroups.com, k8s-infra-oncall@google.com
     description: Runs gencred to refresh generated kubeconfigs.
+# Legacy job for rotating the default sa json key used for legacy prowjobs that
+# can't migrate to use workload identity yet. This json key expires every 90
+# days, so rotating every 15 days should be safe as it allows 5 failures.
+# GCP only allows up to 9 keys for a sa, so deleting old ones on the fly as well.
+- cron: "30 1 1,15 * *"  # At 01:30 on day-of-month 1 and 15.
```

Comment:
`At 01:30 AM, on day 1 and 15 of the month` verified using https://bradymholt.github.io/cron-expression-descriptor/ :)

Reply:
dunno which time zone though :) (but it's ok...)